docs/architecture/0002-routing-mk2.md

```diff
@@ -0,0 +1,109 @@
+# 2. Routing Mk2
```

Comment:
Would maybe a table of contents would be good? With links to each section as I've done in other mds. I know these files aren't that long but could be useful if they became longer maybe? But if you don't think it worth it don't worry

Reply:
The file format is generated by the adr tools so I would rather keep to that.